libMBIN/Source/NMS/Globals/GcVehicleGlobals.cs

```diff
@@ -159,6 +159,7 @@ public class GcVehicleGlobals : NMSTemplate
         /* 0x02E8 */ public GcScanData VehicleLocalScan;
                      [NMS(Size = 0x04, Ignore = true)]
         /* 0x02FC */ public byte[] Padding2FC;
+                     // near match to GcVehicleType BUT Buggy, Bike,..  are reversed
```

Comment:
If you decompile the globals you can see if the enum names are right. It may be that they are incorrectly swapped hare they should actually use the `GcVehicleType` enum

Reply:
I looked in GCVEHICLEGLOBALS.EXML. GcVehicleType should be Bike, Buggy...